.github/workflows/publish_to_maven_local.yml

```diff
@@ -4,12 +4,21 @@ on:
   push:
     branches:
       - main
+    paths:
+      - '**.kt'
+      - '**.kts'
+      - '**.properties'
+      - '**.json'
+      - '**.xml'
+      - '**.yml'
```

Comment:
This could be added to the other workflows as well, specifically the E2E one

Reply:
We could also add `.yaml` and `.conf` here. Just in case.